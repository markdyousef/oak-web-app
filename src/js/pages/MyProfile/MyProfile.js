import React, { Component, PropTypes } from 'react';

import TopNav from '../../components/TopNav';

import css from './MyProfile.css';

class MyProfile extends Component {
    constructor() {
        super();
        this.state = {};
    }
    render() {
        return (
            <div className={css.container}>
                <TopNav />
            </div>
        );
    }
}

export default MyProfile;
