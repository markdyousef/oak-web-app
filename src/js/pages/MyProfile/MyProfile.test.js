// @flow
import React from 'react';
// import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import MyProfile from './MyProfile';

test('renders MyProfile', () => {
    const props = {
        channels: [],
        getTeam: () => {},
        name: 'clai'
    };
    const shallowComponent = shallow(
        <MyProfile {...props} />
    );
    expect(shallowComponent).toMatchSnapshot();
});
