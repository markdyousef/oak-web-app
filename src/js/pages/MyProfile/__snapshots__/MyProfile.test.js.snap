exports[`test renders MyProfile 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className={undefined}>
    <TopNav />
</div>,
  "nodes": Array [
    <div
      className={undefined}>
      <TopNav />
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <MyProfile
        channels={Array []}
        getTeam={[Function]}
        name="clai" />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": MyProfile {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "channels": Array [],
          "getTeam": [Function],
          "name": "clai",
        },
        "refs": Object {},
        "state": Object {},
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className={undefined}>
          <TopNav />
</div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className={undefined}>
          <TopNav />
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <MyProfile
    channels={Array []}
    getTeam={[Function]}
    name="clai" />,
}
`;
